source/clear-linux/guides/network/custom-clear-container.rst

```diff
@@ -0,0 +1,374 @@
+            ...14%
+         Extracting perl-basic pack for version 17790
+            ...28%
+         Extracting openssh-server pack for version 17660
+            ...42%
+         Extracting editors pack for version 17850
+            ...57%
+         Extracting network-basic pack for version 17650
+            ...71%
+         Extracting os-core pack for version 17870
+            ...85%
+         Extracting os-core-update pack for version 17870
+            ...100%
+         Adding any missing files
+            ...88%
+         Inspected 33982 files
+           33974 files were missing
+             33974 of 33974 missing files were replaced
+             0 of 33974 missing files were not replaced
+         Calling post-update helper scripts.
+         WARNING: boot files update skipped due to --no-boot-update argument
+         Fix successful
+
+   .. note::
+
+      The `WARNING` message is expected and can be ignored.
+
+#. Tar up the files and compress it.
+
+   .. code-block:: console
+
+      # tar -C base -cf base.tar .
+      # xz -v -T0 base.tar
+
+#. Create the Dockerfile to build the image.
+
+   .. code-block:: console
+
+      # cat > Dockerfile << EOF
+      FROM scratch
+      MAINTAINER First Last <first.last@example.com>
+      ADD base.tar.xz /
+      CMD ["/bin/bash"]
+      EOF
+
+#. Build the |CL| container image.
+
+   .. code-block:: console
+
+      # docker build -t my-custom-clear-linux-container .
+
+   Example output:
+
+      .. code-block:: console
+
+         # docker build -t my-custom-clear-linux-container .
+
+         Sending build context to Docker daemon  806.5MB
+         Step 1/4 : FROM scratch
+          --->
+         Step 2/4 : MAINTAINER First Last <first.last@example.com>
+          ---> Running in 7238f35abcd0
+          ---> ec5064287c60
+         Removing intermediate container 7238f35abcd0
+         Step 3/4 : ADD base.tar.xz /
+          ---> 2723b7d20716
+         Removing intermediate container 16e3ed0df8da
+         Step 4/4 : CMD /bin/bash
+          ---> Running in efa893350647
+          ---> 5414c3a12993
+         Removing intermediate container efa893350647
+         Successfully built 5414c3a12993
+         Successfully tagged my-custom-clear-linux-container:latest
+
+#. List the newly created |CL| container image.
+
+   .. code-block:: console
+
+      # docker images
+
+   Example output:
+
+      .. code-block:: console
+
+         # docker images
+
+         REPOSITORY                        TAG                 IMAGE ID            CREATED              SIZE
+         my-custom-clear-linux-container   latest              5414c3a12993        About a minute ago   616MB
+
+#. Launch the built |CL| container.
+
+   .. code-block:: console
+
+      # docker run -it my-custom-clear-linux-container
+
+Manage bundles in a Clear Linux based container
+***********************************************
+
+#. To add a bundle to an existing |CL|-based container, use the
```

Comment:
These should not be numbered steps. Rather subsections: Add a bundle and Remove a bundle with the numbered steps on how to do those actions and verify them. Avoid just mentioning actions to be taken or that have been taken and be explicit about how they were performed.